src/main/resources/utilities/populate_default_locus_range_for_species.py

```diff
@@ -0,0 +1,134 @@
+#!/usr/bin/env python3
+
+# Copyright 2019 EMBL - European Bioinformatics Institute
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import click
+import configparser
+import urllib.parse
+
+from pymongo import MongoClient
+from __init__ import *
+
+DEFAULT_LOCUS_RANGE_COLLECTION_NAME = "defaultLocusRange"
```

Comment:
good idea. I assume you plan to make this available through some new webservice to un-hardcode it from the website?

Reply:
Yes. I intend to add a separate end point /default-locus-range similar to /annotation for a given species.